app/views/samples/index.haml

```diff
@@ -36,7 +36,7 @@
                 %td
                   =check_box_tag 'sample_ids[]', sample.id, false, { id: "sample_ids.#{sample.id}" }
                   %label.row{for: "sample_ids.#{sample.id}"}
-                %td= sample.uuid
+                %td{title:sample.uuid}= format_uuid(sample.uuid)
```

Comment:
I think it makes sense to extract the combination of short formatted UUID with full value in the title into another helper method. This is a common pattern. There are probably more locations where a short formatted UUID should be used, and all of them ought to include the full version.
Extracting that into a helper avoids forgetting to add the title on every use. And it allows to easily enhance the component. For example, a feature to copy the full UUID into the clipboard could be useful.